translitua/translit.py

```diff
@@ -112,6 +117,7 @@ class UkrainianKMU(object):
     PATTERN1 = re.compile(u"(?mu)" + u'|'.join(SPECIAL_CASES.keys()))
     PATTERN2 = re.compile(u"(?mu)" + r"\b(" +
                           u'|'.join(FIRST_CHARACTERS.keys()) + u")")
+    DELETE_PATTERN = re.compile(u"(?mu)" + u'|'.join(_DELETE_CASES))
```

Comment:
Erm, I know basic python syntax. Why delete those? Can you show the cases when translation tables aren't enough?